ipv8/requestcache.py

```diff
@@ -154,6 +179,59 @@ def pop(self, prefix, number):
         self.cancel_pending_task(cache)
         return cache
 
+    @contextmanager
+    def passthrough(self,  # pylint: disable=W1113
```

Comment:
I chose to ignore `W1113 (Keyword argument before variable positional arguments list)` here as the Pylint-recommended approach does not allow our required function signature (using `None` as the "no-filter-specified" value). The nastiness here is that the default value for empty variable arguments is an empty tuple, which would be a valid filter to blacklist all classes (whereas `None` whitelists all classes).

The same function signature could be achieved using `functools.singledispatch`, but this would also double the size of this code while making it arguably less readable.
Yet another approach would be to change the function signature to use sentinel objects, which also seems like overkill.